index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200;400;500;700&display=swap" rel="stylesheet">
+</head>
+<body class="main__container">
+    <!-- Creating the navigation menu bar -->
+  <header class="container_menu">
+    <nav class="container--nav">
+        <ul class="navegation--list">
+            <li class="navigation--option"><a>Portfolio</a></li>
+            <li class="navigation--option"><a>About</a></li>
+            <li class="navigation--option"><a>Contact</a></li>
+        </ul>
+        <div class="menu--image">
+            <img src="images\menuUnion.png" alt="Menu button"/>
+        </div>
+    </nav>
+  </header>
+  <!-- This section is the main part, containing heading and a brief description of my work-->
+  <main class="main__containerinfo">
+   <h1 id="titlePortfolio">Hey there. I’m Reynaldo. </h1>
```

Comment:
- You are making use of the semantic `main` tag to represent the main section of the application which is very good practice but however on line 35 you created a `section` outside of this main tag, ideally the headline section which is what contains the main title and information about the developer shoulod be enclosed in a section tag along side the social media icons. for further explanations see the example below of the structure which should be adopted
```html
  <main class="main__containerinfo">
    <section>
      <h1 id="titlePortfolio">Hey there. I’m Reynaldo. </h1>
      <h2 id="developerPortfolio">I’m a software developer</h2>
      <p id="descriptionPortfolio"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
     don’t hesitate to contact me.</p>
      <ul class="container__icons">
        <li class="brand--icon">
          <a href="https://twitter.com/"><img src="images\brand-twitter-filled.png" alt="twitter logo" class="icon--image" /></a>
        </li>
      </ul>
    </section>
   </main>
```